Documentation/app-operations.md

```diff
@@ -0,0 +1,225 @@
+# Fine-Grained Pod Manipulations
+To provide imperative application (application as in application container and the pod being the OS container) 
```

Comment:
Please omit the application container / operating system container reference, I think it confuses things